charts/merlin/values.yaml

```diff
@@ -421,7 +442,8 @@ mlflow:
     # The name of the ServiceAccount to use.
     # If not set and create is true, a name is generated using the fullname template
     name: mlflow
-    annotations: {}
+    annotations:
```

Comment:
Has all of this formatting automatically? Curious, which tool do you use for this?

@caraml-dev/platform, perhaps the repo can enforce the formatting using pre-commit hooks / CI checks.

Reply:
For YAML, I use prettier extension in VS Code.

As for pre-commit, the repo already has: https://github.com/caraml-dev/helm-charts/blob/main/.pre-commit-config.yaml#L12-L15